src/views/SystemList/Table.tsx

```diff
@@ -219,27 +219,18 @@ export const Table = ({
           : -1
     });
 
-    /*
-    {
-      Header: t<string>('systemTable.header.systemStatus'),
-      accessor: 'status',
-      id: 'systemStatus',
-      disableGlobalFilter: true,
-      Cell: ({ row }: { row: Row<CedarSystem> }) => (
-        <div>
-          <SystemHealthIcon
-            status={mapCedarStatusToIcon(row.original.status)}
-            size="medium"
-            className="margin-right-1"
-          />
-          <span>{row.original.status}</span>
-        </div>
-      )
+    if (isHomePage) {
+      cols.push({
+        Header: t<string>('systemTable.header.openRequests'),
+        id: 'openRequests',
+        Cell: ({ row }: { row: Row<CedarSystem> }) =>
+          row.original.linkedSystemIntakes.length +
+          row.original.linkedTrbRequests.length
+      });
```

Comment:
Sorting wont work on this column. Builtin sorting does not go by Cell's return value. To cut to the chase, just overwrite the accessor since there are no other values/props of interest in this column except for the calculated count.
```
      cols.push({
        Header: t<string>('systemTable.header.openRequests'),
        accessor: row => {
          return row.linkedSystemIntakes.length + row.linkedTrbRequests.length;
        }
      });

Reply:
oh nice! thanks Adam - updated